### PR TITLE
feat(docs): Update contribution section of Readme [release skip]

### PR DESCRIPTION
**Pull request by hobbitronics**
- This won't trigger a release

### CODE REVIEW

**Review by Schparky — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -244,4 +244,6 @@ All commit messages must follow the format defined in our [release.config.js fil
...
+These will trigger a patch or minor release depending on the config. A major release can be triggered by including 'BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING' in the footer section followed by ': <description/instructions>' where <description/instructino> details what the breaking change is and/or how to migrate. If this is not done, an empty commit with the proper format can be pushed to trigger the release or by using `git rebase -i` to rewrite the commit messages. commits that contains '[skip release]' or '[release skip]' in their message will be excluded from the commit analysis and won't participate in the release type determination. See https://github.com/semantic-release/semantic-release, https://semantic-release.gitbook.io/semantic-release/support/faq and https://www.conventionalcommits.org/en/v1.0.0/#specification for more details.
```
Comment (mtompset) [inline README.md:247]: Typo in the word instructino. ;)

**Review by mtompset — CHANGES_REQUESTED** [review]
Fix the typo, then it should be good to go.

File: README.md (hunk 1 above)
Comment (hobbitronics) [reply README.md]: Thanks for catching that @mtompset. I finally installed a spell checker plugin to my editor.

**Pull request merged**